aten/src/ATen/native/quantized/cpu/qnnpack/src/q8gemm/4x4c2-dq-sse2.c

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <immintrin.h>
+
+#include <qnnpack/q8gemm.h>
+#include <requantization/runtime-sse2.h>
+
+void pytorch_q8gemm_dq_ukernel_4x4c2__sse2(
+    size_t mr,
+    size_t nr,
+    size_t k,
+    const uint8_t* restrict a,
+    size_t a_stride,
+    const void* restrict w,
+    float* restrict c,
+    size_t c_stride,
+    const union pytorch_qnnp_conv_quantization_params
+        quantization_params[RESTRICT_STATIC 1]) {
```

Comment:
We need to pass the multiplier somehow, either inside pytorch_qnnp_conv_quantization_params or as a separate argument.  Let me know if you have a preference.  Other than that, this should be done.

Reply:
I'd vote for separate argument. See comment below.